### PR TITLE
Improve Solution template

### PR DESCRIPTION
**Pull request by iazehaf**
- [x] Parametrize UI directory 
- [x] Add project name in dash UI
- [x] Add introduction page in dash UI

### CODE REVIEW

**Review by jorgepiloto — APPROVED** [review]
Approving this so the review does not block you, @iazehaf. Apply the code style and feel free to merge when ready. I will perform a patch release once merged.

**Comment by jorgepiloto** [comment]
Is this ready to be merged, @iazehaf?

**Comment by iazehaf** [comment]
@jorgepiloto yes but I'm waiting from one extra tests. I will merge the PR when completed.

**Pull request merged**
